diff -ruN callgrind-0.9.11./configure.in callgrind-0.9.11/configure.in
--- callgrind-0.9.11./configure.in	2005-04-02 13:50:28.000000000 +0200
+++ callgrind-0.9.11/configure.in	2005-04-02 13:54:48.116323688 +0200
@@ -7,19 +7,7 @@
 
 # set default PREFIX / check given PREFIX
 AC_MSG_CHECKING([for valgrind installation])
-if test "x$prefix" = "xNONE"; then
-  vg_exec=`which valgrind`
-  if test "x$vg_exec" = "x"; then
-    AC_MSG_RESULT([not found in $PATH])
-    AC_MSG_ERROR([Please install Valgrind])
-  fi
-else
-  vg_exec=`which $prefix/bin/valgrind`
-  if test "x$vg_exec" = "x"; then
-    AC_MSG_RESULT([not found in $prefix/bin])
-    AC_MSG_ERROR([Valgrind has to be installed in the specified prefix!])
-  fi
-fi
+vg_exec=`which valgrind`
 vg_binpath=`dirname $vg_exec`
 ac_default_prefix=`dirname $vg_binpath`
 if test "x$prefix" = "xNONE"; then
@@ -40,7 +28,7 @@
 CT_OPTIONS="--tool=callgrind"
 use_pkg_config="no"
 AC_MSG_CHECKING([for valgrind version])
-vg_version=`$prefix/bin/valgrind --version`
+vg_version="valgrind-2.4.0"
 case "${vg_version}" in
     valgrind-2.0.*)
 	CT_OPTIONS="--skin=callgrind"
@@ -97,38 +85,6 @@
 AC_MSG_RESULT([${CFLAGS}])
 
 
-# check if there is a vg_skin.h installed
-AC_MSG_CHECKING([if compiling a valgrind tool is possible])
-AC_TRY_COMPILE(
-[#include "vg_skin.h"],
-[],
-[
-# No need for additional directory
-AC_MSG_RESULT([Yes])
-], [
-AC_MSG_ERROR([No. Is a valgrind-dev package missing?])
-])
-
-
-# check if we support skin version of installed valgrind
-#
-# Note that versions used in VG releases 2.y.x, with y odd,
-# are *not* reliable, so we define UNSTABLE to distinguish
-# unstable VG releases from stable ones reusing versions.
-
-AC_MSG_CHECKING([if tool API version is supported])
-AC_TRY_RUN([
-#include "vg_skin.h"
-int main() { return (VG_CORE_INTERFACE_MAJOR_VERSION > 7) ? 1:0; }
-], [
-AC_MSG_RESULT([Yes])
-], [
-AC_MSG_ERROR([Installed valgrind version too new. Please update this package!])
-], [
-AC_MSG_RESULT([Assuming yes (cross-compiling)])
-])
-
-
 AC_OUTPUT(
 	Makefile              \
 	src/Makefile          \
